doc/build.md

```diff
@@ -21,88 +21,108 @@ If `java` is not found, or your version is anything less than `1.8.0_66`, then f
-    $ touch .bashrc
-    $ gedit .bashrc
-    $ export JAVA_HOME=/usr/lib/jvm/java-8-oracle
-    $ echo $JAVA_HOME 
+If `$JAVA_HOME` is not present, open your `.bashrc` file:
+
+    $ touch ~/.bashrc
+    $ gedit ~/.bashrc
+
+For OpenJDK add: `export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64`
+For Oracle JDK add: `export JAVA_HOME=/usr/lib/jvm/java-8-oracle`
+Save and close the file.
+
+Reload the file in your shell:
+
+    $ . ~/.bashrc
+    $ echo $JAVA_HOME
 
 #### 1.2 Other systems
 
-[Download and install the latest JDK]( http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) for your platform.
+[Download and install the latest Oracle JDK]( http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) for your platform.
 
 Build bitcoinj
 -----------------
-### 2. Install bitcoinj fork 
-> _**NOTE:** 
-Bitcoinj versions later than 0.13.1 has removed support for Java serialisation. 
-In version 0.13.1 is also missing support for Java serialisation in MainNetParams (HttpDiscovery.Details).
-We removed usage of Cartographer/HttpDiscovery and fixed privacy issues with Bloom Filters at our [fork version 0.13.1.5](https://github.com/bitsquare/bitcoinj/tree/FixBloomFilters).
-Beside the Java serialisation issues there are [privacy concerns](http://bitcoin-development.narkive.com/hczWIAby/bitcoin-development-cartographer#post3) regarding Cartographer. 
+### 2. Install bitcoinj fork
+> _**NOTE:**
+Bitcoinj versions later than 0.13.1 has removed support for Java serialisation.
+Version 0.13.1 is also missing support for Java serialisation in MainNetParams (HttpDiscovery.Details).
+We removed usage of Cartographer/HttpDiscovery and fixed privacy issues with Bloom Filters in our [fork of version 0.13.1.5](https://github.com/bitsquare/bitcoinj/tree/FixBloomFilters).
+Beside the Java serialisation issues there are [privacy concerns](http://bitcoin-development.narkive.com/hczWIAby/bitcoin-development-cartographer#post3) regarding Cartographer.
 Here is a Github issue with background and open tasks regarding [Bloom Filters](https://github.com/bitsquare/bitsquare/issues/414)._
-                
-    $ git clone -b FixBloomFilters https://github.com/bitsquare/bitcoinj.git               
-    $ cd bitcoinj  
+
+    $ git clone -b FixBloomFilters https://github.com/bitsquare/bitcoinj.git
+    $ cd bitcoinj
     $ mvn clean install -DskipTests -Dmaven.javadoc.skip=true
 
 Prepare Bitsquare build
 -----------------
 
 ### 3. Get Bitsquare source code and build a preliminary Bitsquare version
 
-You need to get the Bitsquare dependencies first as we need to copy the BountyCastle jar to the JRE directory as well as the jdkfix jar. 
+You need to get the Bitsquare dependencies first as we need to copy the BountyCastle jar to the JRE directory as well as the jdkfix jar.
 
     $ git clone https://github.com/bitsquare/bitsquare.git
     $ cd bitsquare
     $ mvn clean package -DskipTests -Dmaven.javadoc.skip=true
```

Comment:
We will update soon to the new JDK (112) once it is out as it has 2 important bugfixes and we can remove then the jdkfix.jar, but I am not sure how long it takes until OpenJDK will have that included as well. But will update build file once that is out (prob. 19.10)...